test/CMakeLists.txt

```diff
@@ -215,21 +231,20 @@ set_tests_properties(test_sqlite_perfdb test_perfdb
 # add_sanitize_test(type_name.cpp)
 
 function(add_custom_test NAME)
-    set(options SKIP_UNLESS_ALL ALLOW_BFLOAT16 ALLOW_HALF ALLOW_INT8 ALLOW_NONXDLOPS)
+    set(options SKIP_UNLESS_ALL ALLOW_BFLOAT16 ALLOW_HALF ALLOW_INT8 ALLOW_NONXDLOPS ALLOW_GEMM_FEATURE)
     set(oneValueArgs)
     set(multiValueArgs)
     cmake_parse_arguments(PARSE "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
     # Many custom tests do test only FP32 data type and therefore
     # added only if none of MIOPEN_TEST_HALF, MIOPEN_TEST_INT8, MIOPEN_TEST_BFLOAT16
     # are set, except the test is allowed explicitly.
-    # Some tests are xDLOPs specific and should not run on gfx900/906 targets.
```

Comment:
Revert this change.

Reply:
I didn't add xDLOPs comments.

Reply:
You've removed them. This removal should be reverted.